sdk/storage/azure-storage-common/inc/azure/storage/common/paged_response.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <cstdlib>
+#include <string>
+
+#include <azure/core/context.hpp>
+#include <azure/core/http/raw_response.hpp>
+
+namespace Azure {
+
+template <class Derived> class PagedResponse {
+public:
+  std::string NextPageToken;
```

Comment:
This field is not supposed to be public; it should only be accessible to the base and derived types.

Reply:
But I think customer may need it in some scenario. like they want to save the token somewhere else and continue listing later.